bgpd/bgp_network.c

```diff
@@ -636,9 +636,6 @@ static int bgp_listener(int sock, struct sockaddr *sa, socklen_t salen,
 	struct bgp_listener *listener;
 	int ret, en;
 
-	sockopt_reuseaddr(sock);
-	sockopt_reuseport(sock);
```

Comment:
We can't do this. Without `SO_REUSEADDR` bgpd might fail on `bind()` if there's a TCP socket from a previous run in the `TIME_WAIT` state (e.g. after restarting bgpd for whatever reason).

Since BGP peers usually try to connect to each other at the same time, bgpd can still work under certain circumstances if it fails to create a listening socket, but relying on this behavior would be wrong.